content/pt-br/docs/tasks/configure-pod-container/extended-resource.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Atribuindo Recursos Estendidos a um Contêiner
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state state="stable" >}}
+
+Esta página mostra como atribuir recursos estendidos a um Contêiner.
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Antes de fazer este exercício, faça o exercício em
+[Anunciar recursos estendidos para um Nó](/docs/tasks/administer-cluster/extended-resource-node/).
+Isso configurará um de seus nós para anunciar um recurso de `dongle`.
```

Comment:
Suggestion: use italic formatting here instead of code formatting (_dongle_ instead of `dongle`).